server/integration/testsuite/src/test/resources/config/infinispan/rolling-upgrades-clustered-7.0.xml

```diff
@@ -1,4 +1,4 @@
-        <subsystem xmlns="urn:infinispan:server:core:8.0" default-cache-container="clustered-new">
+        <subsystem xmlns="urn:infinispan:server:core:7.0" default-cache-container="clustered-new">
```

Comment:
I would run tests against last major ISPN release by default (and in CI), but would keep it if someone wants to test older version so that (s)he has everything prepared and can run tests immediatelly without need to change/understand the source code